zebra-state/src/request.rs

```diff
@@ -401,4 +401,12 @@ pub enum ReadRequest {
     /// * [`Response::Transaction(Some(Arc<Transaction>))`](Response::Transaction) if the transaction is in the best chain;
     /// * [`Response::Transaction(None)`](Response::Transaction) otherwise.
     Transaction(transaction::Hash),
+
+    /// Looks up transactions hashes that were made by provided address in a blockchain height range.
+    ///
+    /// Returns
+    ///
+    /// * A vector of transaction hashes.
+    /// * An empty vector if no transactions were found for the given arguments.
+    TransactionsByAddresses(transparent::Address, block::Height, block::Height),
```

Comment:
Docs: this is a requirement from https://zcash.github.io/rpc/getaddresstxids.html

```suggestion
    ///
    /// Returned txids are in the order they appear in blocks, which ensures that they are topologically sorted
    /// (i.e. parent txids will appear before child txids).
    TransactionsByAddresses(transparent::Address, block::Height, block::Height),
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/4062/commits/b1a69676ca75fed87863b4b6cadcb07225159d34